data/initrd/scripts/prepare_rescue

```diff
@@ -74,6 +74,10 @@ if [ -d /mounts/initrd/update ] ; then
   done
 fi
 
+# create NVMe config files
+nvme gen-hostnqn > /etc/nvme/hostnqn
```

Comment:
Hm, at this point /proc is not mounted and nvme complains a lot - but it does print some string to stdout.

I'm not sure how relevant /proc is for this as this seems to generate just some random string either way. But it's possibly safer to mount and umount /proc around the nvme call. See a few lines below for startshell.

And I see there's also a nvme-gen-hostnqn tool - which doesn't seem to do anything useful at all. But that's OT.

Reply:
I have added mount and umount for /proc.